### PR TITLE
 Create Java code sample for get jobs

### PR DESCRIPTION
**Pull request by chloeliu123**
https://cloud.google.com/dlp/docs/creating-job-triggers

Fixes # Create Java code sample for get jobs

> It's a good idea to open an issue first for discussion.

> [yes] I have followed Sample Format Guide
> [yes] pom.xml parent set to latest shared-configuration
> [yes] Appropriate changes to README are included in PR
> [nothing new] API's need to be enabled to test (tell us)
> [nothing new] Environment Variables need to be set (ask us to set them)
> [yes] Tests pass (mvn -P lint clean verify)
> (Note- Checkstyle passing is required; Spotbugs, ErrorProne, PMD, etc. ERROR's are advisory only)
> [yes] Please merge this PR for me once it is approved

### CODE REVIEW

**Comment by lesv** [comment]
@chloeliu123 If you add yourself to the gcp team at `github/chloeliu123`, tests will auto run for you.

File: dlp/src/test/java/dlp/snippets/JobsTests.java [hunk 1]
```diff
@@ -106,6 +106,18 @@ public void testCreateJobs() throws Exception {
     assertThat(output, CoreMatchers.containsString("Job created successfully."));
   }
 
+  @Test
+  public void testGetJobs() throws Exception {
+    // Create a job with a unique UUID to be gotten
+    String jobId = UUID.randomUUID().toString();
+    createJob(jobId);
```
Comment (lesv) [inline dlp/src/test/java/dlp/snippets/JobsTests.java:113]: Great - should this be deleted at some point as these tests run many times / day?

Comment (chloeliu123) [reply dlp/src/test/java/dlp/snippets/JobsTests.java]: > Great - should this be deleted at some point as these tests run many times / day?

Thanks for bringing this up, I've updated the get and create tests to delete the created instance after the tests are done. To do that, I also made a minor change to JobsCreate.java

**Comment by lesv** [comment]
Thank you!

**Pull request merged**